.gitignore

```diff
@@ -3,6 +3,9 @@ composer.lock
 *.sh
 script/*
 
+# IDE
```

Comment:
This is not required, you should be able to add IDE specific ignores to your global .gitignore and not the projects gitignore.